helpers/test/focus.test.js

```diff
@@ -144,11 +150,18 @@ describe('focus', () => {
 				.to.equal(elem.querySelector('#light1'));
 		});
 
+		it('returns focusable not tabblable child', async() => {
+			const elem = await fixture(focusableNotTabbableFixture);
+			expect(getFirstFocusableDescendant(elem.getContent(), false, true, false))
```

Comment:
Looks like this should just be `elem` instead of `elem.getContent()`...? Same with the other test.